scriptworker/cot/verify.py

```diff
@@ -31,6 +34,18 @@
 log = logging.getLogger(__name__)
 
 
+DECISION_MACH_COMMANDS = ((
+    './mach', 'taskgraph', 'decision'
+), (
+    './mach', 'taskgraph', 'add-tasks'
+), (
+    './mach', 'taskgraph', 'backfill'
+), (
+    './mach', 'taskgraph', 'action-callback'
```

Comment:
Question: I see what the other commands are for. What is this one?

Reply:
I think it's the new way of doing things. https://firefox-source-docs.mozilla.org/taskcluster/taskcluster/actions.html#creating-a-callback-action